src/ai/backend/manager/models/group.py

```diff
@@ -193,6 +196,48 @@ async def resolve_groups(
     return return_val
 
 
+async def query_suadmin_accessible_groups(
+    db_conn: SAConnection,
+) -> None:
+    """
+    Query all groups.
+    This function returns `None` since super admin can access all groups
+    and fetching all groups from DB is a big overhead.
+    """
+    return None
+
+
```

Comment:
Is there a reason to leave non-functioning function in code?

Reply:
I want `query_accessible_groups()` method for all User roles. This function does not do anything but it seems normalized and readable code for me. How do you think? I will remove it if it does not make sense.

Reply:
Adding comment that indicates superadmin can access all groups in place of this empty function seems enough.